Base/usr/share/man/man1/Magnifier.md

```diff
@@ -0,0 +1,21 @@
+## Name
+
+![Icon](/res/icons/16x16/app-magnifier.png) Magnifier - Magnifier application
+
+[Open](file:///bin/Magnifier)
+
+## Synopsis
+
+```**sh
+$ Magnifier
+```
+
+## Description
+
+`Magnifier` is an application that magnifies or zooms-in an area of the desktop where the mouse cursor hovers.
+
+You can launch _Magnifier_ by selecting Serenity menu -> Utilities -> Magnifier.
```

Comment:
I don't know if they should, but other man pages don't tell how to open GUI applications.

Reply:
Magnifier is buried 2 levels deep and giving heads up on how to find it can be quite useful.